### DISCUSSION TITLE
Hovering over the issue marker doesn't display its tooltip with the issue description

### DISCUSSION THREAD

**Discussion by rhumbertgz** [eclipse-glsp/glsp#1099] [Q&A] [answered]
I was following the documentation for [GIssueMarkerView](https://www.eclipse.dev/glsp/documentation/rendering/) and ran into the problem that "builder() does not exist on type GIssueMarker". Fortunately, I found a solution in this [discussion thread](https://github.com/eclipse-glsp/glsp/discussions/772#discussioncomment-4141485). However, although the icon is shown now, the tooltip is not displayed when hovering. I have tried both node template examples. For vscode I'm using a dev container since the gasp-server crashes (see issue [#1095](https://github.com/eclipse-glsp/glsp/discussions/1095#discussion-5550219)). For Theia, I'm using the dev container provided in the `node-json-theia` template. I also tried multiple browsers (Safari, Chrome, Edge) and I get the same error. 

![Aug-28-2023 22-47-42](https://github.com/eclipse-glsp/glsp/assets/10393245/a96a4df4-5205-4501-819a-b19e0e34892e)

Here is my the updated `createTaskNode` method defined  at `/node-json-theia/glsp-server/src/model/tasklist-gmodel-factory.ts`

 ```
protected createTaskNode(task: Task): GNode {
        const builder = GNode.builder()
            .id(task.id)
            .addCssClass('tasklist-node')
            .add(GLabel.builder().text(task.name).id(`${task.id}_label`).build())
            .layout('hbox')
            .addLayoutOption('paddingLeft', 5)
            .position(task.position);

        builder.add(new GIssueMarkerBuilder(GIssueMarker)
            .type(DefaultTypes.ISSUE_MARKER)
            .addIssue({ message: 'Information message', severity: 'info' })
            .position(-8, -8)
            .build()
        );

        if (task.size) {
            builder.addLayoutOptions({ prefWidth: task.size.width, prefHeight: task.size.height });
        }

        return builder.build();
    }
```

I also updated the `taskListDiagramModule` defined at `/workspaces/node-json-theia/glsp-client/tasklist-glsp/src/di.config.ts` as described in the documentation

```
const taskListDiagramModule = new ContainerModule((bind, unbind, isBound, rebind) => {
    rebind(TYPES.ILogger).to(ConsoleLogger).inSingletonScope();
    rebind(TYPES.LogLevel).toConstantValue(LogLevel.warn);
    const context = { bind, unbind, isBound, rebind };
    configureDefaultModelElements(context);
    configureModelElement(context, DefaultTypes.LABEL, SLabel, SLabelView, { enable: [editLabelFeature] });
    configureModelElement(context, DefaultTypes.ISSUE_MARKER, SIssueMarker, GIssueMarkerView);
});
```

Any suggestion?

**Comment by tortmayr** [comment, marked as answer]
Hi @rhumbertgz,

I had a quick look at this and it seems like the documentation is outdated here. The hover mouse listener only displays the tooltip if the target element is an instance of `GIssueMarker`. So if you change your `configureModelElement` call in the `taskListDiagramModule` to
```ts
configureModelElement(context, DefaultTypes.ISSUE_MARKER, GIssueMarker, GIssueMarkerView);
```
it should work. We will update the documentation accordingly. Sorry fore the inconveniences.

**Reply by rhumbertgz** [reply]
Thanks @tortmayr. Everything works now 👍🏼
